core/domain/wipeout_service.py

```diff
@@ -1241,26 +1345,22 @@ def _pseudonymize_models_transactional(
             if general_suggestion_model.final_reviewer_id == user_id:
                 general_suggestion_model.final_reviewer_id = pseudonymized_id
             general_suggestion_model.update_timestamps()
-
-        datastore_services.put_multi(
-            feedback_thread_models +
-            feedback_message_models +
-            general_suggestion_models)
+        datastore_services.put_multi(general_suggestion_models)
 
     feedback_ids_to_pids = (
         pending_deletion_request.pseudonymizable_entity_mappings[
             models.NAMES.feedback.value])
     for feedback_id, pseudonymized_id in feedback_ids_to_pids.items():
-        feedback_related_models = [
+        feedback_related_models: List[base_models.BaseModel] = [
```

Comment:
I'm a bit confused about why mypy can't handle the old code. I mean ... at the end of the process in the new code you end up with a list with multiple types anyway, right, so shouldn't mypy have problems with the new code as well?

Reply:
Here we know that ' In this process ' we end up having a list with multiple types ( because i think it is a special case ), But MyPy will throw an error considering it as a normal case like others.
for eg:
```python
L1: List[str] = ['a', 'b']
L2: List[int] = [1, 2]

L1 + L2     # MyPy will throw an error. 
```
Also can you please be more specific about old_code and new_code.

Reply:
By old code I mean the original code in this file before you modified it. I get the issue with different types but wouldn't they all still be subclasses of BaseModel (say)?

Reply:
Yes, they all are subclasses of BaseModel and that's the reason why we are able to combine all models inside in one list ( by defining `feedback_related_models`  as a list of BaseModels )

Ignore is Removed!